pom.xml

```diff
@@ -864,11 +864,6 @@
                 <artifactId>curator-recipes</artifactId>
                 <version>${curator.version}</version>
             </dependency>
-            <dependency>
```

Comment:
https://github.com/apache/incubator-seatunnel/blob/dev/tools/dependencies/known-dependencies.txt#L110  we used the curator-recipes, why did we need to delete this? The fixed version is a very important thing when there are multiple versions.

Reply:
> https://github.com/apache/incubator-seatunnel/blob/dev/tools/dependencies/known-dependencies.txt#L110 we used the curator-recipes, why did we need to delete this? The fixed version is a very important thing when there are multiple versions.

@CalvinKirs because there is still a same dependency above at line 862 🙈

Reply:
Sorry, @ljl1988com my fault